index.js

```diff
@@ -47,14 +51,15 @@ module.exports = (target, opts) => {
 			args = args.concat(appArgs);
 		}
 	} else {
+		if (appArgs.length > 0) {
+			args = args.concat(appArgs);
+		}
+
 		if (opts.app) {
 			cmd = opts.app;
 		} else {
-			cmd = path.join(__dirname, 'xdg-open');
-		}
-
-		if (appArgs.length > 0) {
-			args = args.concat(appArgs);
+			cmd = path.join('/usr', 'bin', 'env');
```

Comment:
I'd use something like [which](https://github.com/npm/node-which) here to ensure that we don't use the wrong path.